swo_aws_extension/flows/jobs/process_aws_invitations.py

```
@@ -0,0 +1,113 @@
+                )
+                continue
+
+            logger.info(
+                "%s - Skip - Billing transfer invitation %s is still in REQUESTED status. "
+                "Will check again later.",
+                context.order_id,
+                transfer_id,
+            )
+
+    # TODO: SDK candidate
+    def _get_querying_orders(self):  # noqa: WPS210
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `WPS210`)

Remove unused `noqa` directive

(RUF100)